src/ast/ast.h

```diff
@@ -69,4 +96,16 @@ namespace ast
     iteration_parent() = prev_parent;
     iteration_index() = prev_index;
   }
+
+  extern "C"
+  {
+    /// This method is for debug only,
+    /// when needing to dump ast trees inside a debugger.
+    /// This is needed because `peg::ast_to_s` is templated.
+    /// Return 0 to make gdb happy.
+    /// FIXME: This does not work on LLDB:
+    ///   error: Couldn't lookup symbols
+    ///     ast::dump(std::shared_ptr<peg::AstBase<ast::Annotation> > const&)
+    size_t dump(const ::ast::Ast& ast);
```

Comment:
As we talked about, perhaps `::ast_dump` will work in LLDB.

Reply:
It does, updated now.